src/vos/vos_dtx.c

```diff
@@ -1732,7 +1775,12 @@ vos_dtx_check(daos_handle_t coh, struct dtx_id *dti, daos_epoch_t *epoch,
 
 	d_iov_set(&kiov, dti, sizeof(*dti));
 	d_iov_set(&riov, NULL, 0);
-	rc = dbtree_lookup(cont->vc_dtx_active_hdl, &kiov, &riov);
+
```

Comment:
adding a few wrapper functions for these tree operations could make the code more cleaner.

Reply:
I will merge solo and active tree together to cleanup the code.